tests/aaa_test.py

```diff
@@ -5,29 +5,27 @@
 import pathlib
 from honeybee_ifc.model import Model
 from honeybee.model import Model as HBModel
+import pytest
 
-file_1 = pathlib.Path('tests/assets/ifc/FamilyHouse_AC13.ifc')
-file_2 = pathlib.Path('tests/assets/ifc/SmallOffice_d_IFC2x3.ifc')
+# file_1 = pathlib.Path('tests/assets/ifc/FamilyHouse_AC13.ifc')
+# file_2 = pathlib.Path('tests/assets/ifc/SmallOffice_d_IFC2x3.ifc')
 
 
-def test_assets():
-    """Test that the files in assets remain."""
-    assert file_1.stem == 'FamilyHouse_AC13'
-    assert file_2.stem == 'SmallOffice_d_IFC2x3'
+# def test_assets():
+#     """Test that the files in assets remain."""
+#     assert file_1.stem == 'FamilyHouse_AC13'
+#     assert file_2.stem == 'SmallOffice_d_IFC2x3'
 
-
-def test_convert_to_hbjson():
+def test_convert_to_hbjson(tmp_path: pathlib.Path,
```

Comment:
@devngc, just FYI, `pytest` has a built-in `tmp_path` [fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html) which will handle creating/cleaning temp files for you in test suites.

Reply:
Good to know. I am not sure if it would work best in this case. You will see that I have aaa_test, zzz_test and some other tests. Translating IFC to HBJSON takes more than a minute depending on the file and the number of CPUs. Therefore, I make sure this translation happens in aaa_test. Rest of the tests use these translated HBJSON and lastly, zzz_test deletes the whole temp folder. 

Correct me if I am wrong but the scope of tmp_path fixture is local to a particular unit test.

Reply:
Fixtures can use the `scope` kwarg to control their lifetime. In this case, I think the `'session'` value could be used to create a temp dir that lasts for the duration of the test suite and is removed at the end.

Reply:
So you were right, the `tmp_path` built-in fixture is function scoped. But, I made a new one using `tempfile` which achieves a similar result. I also moved the `house_model` and `office_model` fixtures to the `conftest` as session-scoped as recreating them seemed to be making the tests take longer than necessary.